tests/topotests/lib/common_config.py

```diff
@@ -743,8 +743,10 @@ def start_topology(tgen, daemon=None):
             TopoRouter.RD_ZEBRA, "{}/{}/zebra.conf".format(TMPDIR, rname)
         )
 
-        # Loading empty bgpd.conf file to router, to start the bgp daemon
-        router.load_config(TopoRouter.RD_BGP, "{}/{}/bgpd.conf".format(TMPDIR, rname))
+        # Loading empty bgpd.conf file to router, to start the bgp deamon
```

Comment:
Why this whitespace change?

Reply:
It is resolved, changes are merged via different PR.